src/Components/Classes/Form/form.module.css

```diff
@@ -46,10 +46,15 @@
 
 .buttonsDiv {
   display: flex;
+  flex-direction: column;
   justify-content: center;
   align-items: center;
 }
 
+
```

Comment:
great job Fer!!!, if you have to make a new commit for another correction take the opportunity to delete these blank lines.